MapboxVisionFleet/VisionFleetManager/VisionFleetManager.swift

```diff
@@ -0,0 +1,18 @@
+import Foundation
+import MapboxVision
+import MapboxVisionFleetNative
+import MapboxVisionSafety
+
+public final class VisionFleetManager {
```

Comment:
there should be sendEvent method or it should be removed from native

Reply:
See my previous comment.